src/CPK.ts

```diff
@@ -219,59 +245,59 @@ class CPK {
   async execTransactions(
     transactions: Transaction[],
     options?: ExecOptions
-  ): Promise<TransactionResult> {
-    if (!this.transactionManager) {
-      throw new Error('CPK transactionManager uninitialized')
-    }
-
+  ): Promise<TransactionResult | void> {
```

Comment:
Can actually remove `| void` from the type here. In the long run, I would consider merging the `safeAppsSdkConnector` and `safeAppsSdkTransactionManager` though.